cla_backend/apps/checker/views.py

```diff
@@ -53,6 +54,9 @@ class EligibilityCheckViewSet(
 ):
     serializer_class = EligibilityCheckSerializer
 
+    def get_request_user(self):
```

Comment:
you should return the user ;-)

Also, there's already a util func for getting the web user `get_web_user` already imported in this file